packages/lab/src/calendar/internal/CalendarDay.tsx

```diff
@@ -56,9 +58,10 @@ export const CalendarDay = forwardRef<HTMLButtonElement, CalendarDayProps>(
         {
           [withBaseName("hidden")]: hidden,
           [withBaseName("outOfRange")]: outOfRange,
-          [withBaseName("unselectableHighEmphasis")]: unselectable === "high",
           [withBaseName("today")]: today,
-          [withBaseName("unselectableLowEmphasis")]: unselectable === "low",
+          [withBaseName("unselectable")]: !!unselectable,
+          uitkLowEmphasis: unselectable === "low",
```

Comment:
are these needed if the classes don't do anything? (if so, also switch to uitkEmphasisLow here)

Reply:
I think it's fine to keep them. Low emphasis is used for styling within CSS but medium isn't. We could remove medium but a red cross is added in medium but via the dom.